chord_metadata_service/chord/tests/test_api_data_types.py

```diff
@@ -1,18 +1,77 @@
+import uuid
+
 from django.urls import reverse
+from django.test import TestCase
 from rest_framework import status
-from rest_framework.test import APITestCase
+
+from chord_metadata_service.authz.tests.helpers import AuthzAPITestCase, PermissionsTestCaseMixin
+from chord_metadata_service.discovery.tests.constants import DISCOVERY_CONFIG_TEST
+from chord_metadata_service.patients import models as pa_m
+from chord_metadata_service.phenopackets.tests import constants as ph_c
+from chord_metadata_service.phenopackets import models as ph_m
 
 from ..data_types import DATA_TYPE_EXPERIMENT, DATA_TYPE_PHENOPACKET, DATA_TYPES
+from ..models import Project, Dataset
 from ..views_data_types import get_count_for_data_type
+from .constants import VALID_DATA_USE_1
 
 POST_GET = ("POST", "GET")
 
 DATA_TYPE_NOT_REAL = "not_a_real_data_type"
 
 
-class DataTypeTest(APITestCase):
+class DataTypeHelperTest(TestCase, PermissionsTestCaseMixin):
+    def setUp(self):
```

Comment:
Can extend `PhenoTestCase` class here to factor out this setup code

Reply:
done